### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by banlon-jones**
updated queries.sql file
updated data.sql file
updated schema.sql file

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -39,3 +39,16 @@ select neutered, max(escape_attempts) from animals group by neutered;
...
+
+SELECT animals.name, owners.fulL_name from owners LEFT JOIN animals on owners.id = animals.owners_id;
+
+SELECT species.name, COUNT(*) from animals JOIN species ON species.id = animals.species_id GROUP BY species.name;
+
+SELECT animals.name, owners.full_name, species.name from animals JOIN species ON species.id = animals.species_id JOIN owners ON owners.id = animals.owners_id WHERE species.name = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
+
+SELECT * from animals JOIN owners ON animals.owners_id = owners.id WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
+
+SELECT owners.full_name, COUNT(animals.name) from owners LEFT JOIN animals ON owners.id = animals.owners_id GROUP BY owners.full_name ORDER BY num_animals DESC;
```
Comment (DammyShittu) [inline queries.sql]: - While trying to run this query, I ran into an error with `num_animals` as it wasn't present anywhere in the query.
Here 👇 is a screenshot of the output.



I believe the correct query should be like this 👇 

```
SELECT owners.full_name, COUNT(animals.name) from owners 
LEFT JOIN animals ON owners.id = animals.owner_id 
GROUP BY owners.full_name
ORDER BY COUNT(animals.name) DESC;
```

Kindly fix it. 😉

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @banlon-jones, 👋 

You have done an amazing job so far. 

Well done 👍 

There is a little issue that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

_Check the comment under the review._


# Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (banlon-jones) [reply queries.sql]: Thanks for the feedback and correction really appreciate it :+1:  fixed

**Review by descholar-ceo — APPROVED** [review]
Hi @banlon-jones , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**